doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -39,44 +39,44 @@ nRF5340 DK RAM and flash memory requirements
 
 The following tables present memory requirements for samples running on the :ref:`nRF5340 DK <gs_programming_board_names>` (:ref:`nrf5340dk_nrf5340 <zephyr:nrf5340dk_nrf5340>`) with the software cryptography support provided by the :ref:`nrfxlib:nrf_oberon_readme` module.
 
-.. table:: nRF5340 multiprotocol Thread 1.2 memory requirements
+.. table:: nRF5340 single protocol Thread 1.3 memory requirements
```

Comment:
Just confirming: you're inverting the order of tables? I wouldn't want this to be some typo :)

Reply:
Yes, exactly, the order was inverted :)